tox.ini

```diff
@@ -2,9 +2,10 @@
 envlist =
     py{27,34,35,36,37,38}-default
```

Comment:
Should we add py39 to this and below test envs as well?

Reply:
Done